images/multus-daemonset.yml

```diff
@@ -190,6 +190,40 @@ spec:
           mountPath: /host/opt/cni/bin
         - name: multus-cfg
           mountPath: /tmp/multus-conf
+      initContainers:
+        - name: install-multus-binary
+          image: ghcr.io/k8snetworkplumbingwg/multus-cni:stable
+          command:
+            - "cp"
+            - "/usr/src/multus-cni/bin/multus"
+            - "/host/opt/cni/bin/multus"
+          resources:
+            requests:
+              cpu: "10m"
+              memory: "15Mi"
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - name: cnibin
+              mountPath: /host/opt/cni/bin
+              mountPropagation: Bidirectional
+        - name: generate-kubeconfig
+          image: docker.io/maiqueb/multus:latest
```

Comment:
can we use `ghcr.io/k8snetworkplumbingwg/multus-cni:stable`?

Reply:
sure, this is just me being lazy and not using a local registry :) 

I'll update before I un-draft the PR.

Reply:
Another thing that comes to mind is we won't be able to update this in one go: unless we immediately cherry-pick this PR to stable - which I don't recommend :) 

I guess for now we should just make sure the e2e part is working correctly, and later on, (after this is cherry-picked to stable) on a separate PR, update the install script.

I also have some conflicting ideas about non-kubernetes installations ... The entry point script would be needed for that. Is that something we want to keep supporting ? @s1061123 / @dougbtv

Reply:
We've discussed this offline; the plan is:
- issue a multus release before this PR - and all other thick plugin work.
- keep referencing in the official installation this entrypoint based script until the thick plugin approach matures enough to enter the `stable` branch.
- once the thick plugin approach is mature enough, update the multus-daemonset yaml file to reflect the new installation type.